src/operations/client_bulk_write/executor.ts

```diff
@@ -31,9 +33,13 @@ export class ClientBulkWriteExecutor {
     operations: AnyClientBulkWriteModel[],
     options?: ClientBulkWriteOptions
   ) {
+    if (operations.length === 0) {
+      throw new MongoClientBulkWriteExecutionError('No client bulk write models were provided.');
+    }
+
     this.client = client;
     this.operations = operations;
-    this.options = { ...options };
+    this.options = { ordered: true, ...options };
```

Comment:
Do we need to apply defaults for bypassDocumentValidation or verboseResults too?

Reply:
Add defaults for both.